ansible/roles/connection_db/handlers/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Restart Redis
+  service:
+    name: "redis"
+    state: restarted
+    enabled: yes
+  delegate_to: "{{ connection_db_host }}"
+  become: True
+
+- name: Sleep
+  pause:
+    seconds: 10
+
+- name: Init connection db
+  debug:
+    msg: >
+      Run 'init_connection_db'
+      {{ lookup('servercfgd_client', 'init_connection_db', servercfgd_host=connection_db_host_ip) }}
```

Comment:
Is this to call a function via xmlrpc? 
Generally `lookup` is for getting some information from somewhere. Using `lookup` to perform RPC call does not seem like an intuitive way.
Ever considered running customized ansible module for these tasks?

Reply:
Yes, but one reason that I favor `Lookup` over Ansible modules is that it runs natively on the control machine without copying to the remote(`localhost` in this case).

Reply:
But eventually it is executed on the remote machine through the XMLRPC TCP port exposed at 10003.

Reply:
Yes, but what matters most here is where do we set up the RPC client?
For modules: copy to the dest(localhost) -> call rpc server
For lookups: call rpc server
From the [plugin doc](https://docs.ansible.com/ansible/latest/plugins/lookup.html), plugins are used to access data from outside resources(files, databases, key/value stores, APIs, and other services). IMHO, rpc calls are like those outside service resources for `Ansible`.

Reply:
For modules, copy to the dest, then run the module to setup DB. There is no need to call RPC. Right? What I am suggesting is not to use RPC at all. Introducing RPC is like introducing another way of remote execution which is already available by customized ansible modules. So, is there something that can only be done by RPC or is there special advantage of RPC against customized ansible module?

Reply:
After offline discussion, the servercfgd RPC server is needed for future use. Then it makes sense to run local plugin to access the APIs exposed by servercfgd. LGTM.